src/test/java/com/bakdata/common_kafka_streams/integration/DeleteTopicTest.java

```diff
@@ -61,16 +63,19 @@ void setup() {
         this.kafkaCluster.start();
 
         final Properties kafkaProperties = new Properties();
-        kafkaProperties.put("application.id", "id");
-        kafkaProperties.put("bootstrap.servers", this.kafkaCluster.getBrokerList());
+        kafkaProperties.put(StreamsConfig.APPLICATION_ID_CONFIG, "id");
+        kafkaProperties.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, this.kafkaCluster.getBrokerList());
 
+        final StreamsBuilder builder = new StreamsBuilder();
+        builder.stream("input").to("output");
+        final Topology topology = builder.build();
```

Comment:
Couldn't you write the lines 69-71 in one line?

Reply:
No, to returns void